.drone.yml

```diff
@@ -625,4 +643,392 @@ matrix:
+      FEDERATION_OC_VERSION: daily-stable10-qa
+      TEST_SUITE: api
+      ENCRYPTION_TYPE: masterkey
+      DB_TYPE: mysql
+      DB_HOST: mysql
+      PART: 8
+
+    ## API core stable10 with user-keys encryption
+    - PHP_VERSION: 5.6
+      OC_VERSION: daily-stable10-qa
+      PREPARE_ACCEPTANCE: true
+      USE_SERVER: true
+      USE_FEDERATED_SERVER: true
+      FEDERATION_OC_VERSION: daily-stable10-qa
+      TEST_SUITE: api
+      ENCRYPTION_TYPE: user-keys
+      DB_TYPE: mysql
+      DB_HOST: mysql
+      PART: 1
+
+    - PHP_VERSION: 5.6
+      OC_VERSION: daily-stable10-qa
+      PREPARE_ACCEPTANCE: true
+      USE_SERVER: true
+      USE_FEDERATED_SERVER: true
+      FEDERATION_OC_VERSION: daily-stable10-qa
+      TEST_SUITE: api
+      ENCRYPTION_TYPE: user-keys
+      DB_TYPE: mysql
+      DB_HOST: mysql
+      PART: 2
+
+    - PHP_VERSION: 5.6
+      OC_VERSION: daily-stable10-qa
+      PREPARE_ACCEPTANCE: true
+      USE_SERVER: true
+      USE_FEDERATED_SERVER: true
+      FEDERATION_OC_VERSION: daily-stable10-qa
+      TEST_SUITE: api
+      ENCRYPTION_TYPE: user-keys
+      DB_TYPE: mysql
+      DB_HOST: mysql
+      PART: 3
+
+    - PHP_VERSION: 5.6
+      OC_VERSION: daily-stable10-qa
+      PREPARE_ACCEPTANCE: true
+      USE_SERVER: true
+      USE_FEDERATED_SERVER: true
+      FEDERATION_OC_VERSION: daily-stable10-qa
+      TEST_SUITE: api
+      ENCRYPTION_TYPE: user-keys
+      DB_TYPE: mysql
+      DB_HOST: mysql
+      PART: 4
+
+    - PHP_VERSION: 5.6
+      OC_VERSION: daily-stable10-qa
+      PREPARE_ACCEPTANCE: true
+      USE_SERVER: true
+      USE_FEDERATED_SERVER: true
+      FEDERATION_OC_VERSION: daily-stable10-qa
+      TEST_SUITE: api
+      ENCRYPTION_TYPE: user-keys
+      DB_TYPE: mysql
+      DB_HOST: mysql
+      PART: 5
+
+    - PHP_VERSION: 5.6
+      OC_VERSION: daily-stable10-qa
+      PREPARE_ACCEPTANCE: true
+      USE_SERVER: true
+      USE_FEDERATED_SERVER: true
+      FEDERATION_OC_VERSION: daily-stable10-qa
+      TEST_SUITE: api
+      ENCRYPTION_TYPE: user-keys
+      DB_TYPE: mysql
+      DB_HOST: mysql
+      PART: 6
+
+    - PHP_VERSION: 5.6
+      OC_VERSION: daily-stable10-qa
+      PREPARE_ACCEPTANCE: true
+      USE_SERVER: true
+      USE_FEDERATED_SERVER: true
+      FEDERATION_OC_VERSION: daily-stable10-qa
+      TEST_SUITE: api
+      ENCRYPTION_TYPE: user-keys
+      DB_TYPE: mysql
+      DB_HOST: mysql
+      PART: 7
+
+    - PHP_VERSION: 5.6
+      OC_VERSION: daily-stable10-qa
+      PREPARE_ACCEPTANCE: true
+      USE_SERVER: true
+      USE_FEDERATED_SERVER: true
+      FEDERATION_OC_VERSION: daily-stable10-qa
+      TEST_SUITE: api
+      ENCRYPTION_TYPE: user-keys
+      DB_TYPE: mysql
+      DB_HOST: mysql
+      PART: 8
+
+    ## API core master with masterkey encryption
+    - PHP_VERSION: 7.1
+      OC_VERSION: daily-master-qa
+      PREPARE_ACCEPTANCE: true
+      USE_SERVER: true
+      USE_FEDERATED_SERVER: true
+      FEDERATION_OC_VERSION: daily-stable10-qa
```

Comment:
still says stabe10 here - 8 parts

Reply:
that is the federation server. currently the fed. servers in all tests is running stable10. Encryption is not enabled on the fed. server and we want to make sure that any change can still talk to an existing server in the wild. stable10 is the closest we can get to a possible version in the wild

Reply:
Good, glad you are thinking :)